Formula/libslax.rb

```diff
@@ -20,27 +21,53 @@ class Libslax < Formula
     sha256 high_sierra:    "95e8b6bdc7010103110d8c7a92c33dd8e2e04228e037ca81c3a5cb69ea955ab2"
   end
 
-  head do
-    url "https://github.com/Juniper/libslax.git"
-
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-  end
-
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
   depends_on "libtool" => :build
   depends_on "openssl@1.1"
 
+  uses_from_macos "bison" => :build
+  uses_from_macos "curl"
+  uses_from_macos "libedit"
+  uses_from_macos "libxml2"
+  uses_from_macos "libxslt"
+  uses_from_macos "sqlite"
+
   conflicts_with "genometools", because: "both install `bin/gt`"
   conflicts_with "libxi", because: "both install `libxi.a`"
 
+  # Fix compilation in bison 3.7.6+. Patch accepted upstream, remove on next release
+  patch do
+    url "https://github.com/Juniper/libslax/commit/cc693df657bc078cd11abe910cbb94ce2acaed67.patch?full_index=1"
+    sha256 "68cdafb11450cd07bdfd15e5309979040e5956c3e36d9f8978890c29c8f20e87"
+  end
+
+  # Fix detection of libxml2 in configure. Two following patches accepted upstream, remove on next release
+  patch do
+    url "https://github.com/Juniper/libslax/commit/5fda392d357b753f7e163f94b8795c028300b024.patch?full_index=1"
+    sha256 "0a424f900e76faa8f1f1c7de282455d1b77c402329a6dc0be7e6370e9aa790de"
+  end
+
+  patch do
+    url "https://github.com/Juniper/libslax/commit/c1b0ba1a342bd4f1ee58f8a339cbd29938d58ba9.patch?full_index=1"
+    sha256 "fa5ecd56672843838cef62d7d44520613c5fa0e5904e3497266d0ee45b16df04"
+  end
+
   def install
-    # configure remembers "-lcrypto" but not the link path.
-    ENV.append "LDFLAGS", "-L#{Formula["openssl@1.1"].opt_lib}"
+    # Upstream patches have already bumped package version in configure.ac ahead of new release being tagged;
+    # remove on next release
+    inreplace "configure.ac",
+              "AC_INIT([libslax],[0.22.2],[phil@juniper.net])",
+              "AC_INIT([libslax],[#{version}],[phil@juniper.net])"
+
+    # Upstream setup.sh script will not overwrite/replace an existing configure script
+    rm "configure"
+    system "sh", "./bin/setup.sh"
```

Comment:
Could `autoreconf --force --verbose --install` work here instead?  It looks like the `setup.sh` script just runs `autoreconf --install` which won't overwrite the old `configure` script as you've noted.